apps/dashboard/app/helpers/system_status_helper.rb

```diff
@@ -26,6 +39,10 @@ def percent(value, total)
     "#{(value.to_f / total * 100).round 2}%"
   end
 
+  def valid_percent(percent)
+    percent.to_f >= 0
+  end
```

Comment:
Can we change this to have a `?` in it like `valid_percent?`. That just makes it read a bit easier and you already sort of know the return value as a Boolean given the convention.